pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -118,18 +114,15 @@ pub mod pallet {
 		/// The sender account that will be used in the OutboundQueue
 		/// implementation.
 		#[pallet::constant]
-		type Sender: Get<Self::AccountId>;
+		type Sender: Get<DomainAddress>;
```

Comment:
If instead of `AccountId` we can use `DomainAddress::centrifuge(AccountId)`, which is isometric, it would work better with the internals for sending messages.